dev/bots/prepare_package.dart

```diff
@@ -833,16 +833,8 @@ class ArchivePublisher {
       print('gsutil.py -- $args');
       return '';
     }
-    if (platform.isWindows) {
-      return _processRunner.runProcess(
-        <String>['python3', path.join(platform.environment['DEPOT_TOOLS']!, 'gsutil.py'), '--', ...args],
-        workingDirectory: workingDirectory,
-        failOk: failOk,
-      );
-    }
-
     return _processRunner.runProcess(
-      <String>['gsutil.py', '--', ...args],
+      <String>['python3', path.join(platform.environment['DEPOT_TOOLS']!, 'gsutil.py'), '--', ...args],
```

Comment:
so DEPOT_TOOLS env var is set on linux and mac, right? previously we were relying on it being on path

Reply:
That's correct.

Reply:
cool, lgtm